faq.md

```diff
@@ -1,45 +1,634 @@
+
+- **OpenTK 3.x** contains its own WinForms-compatible `GLControl` component.
+
+### How fast is OpenTK?
+
+For a .NET library, OpenTK is *very* fast.  OpenTK 3 and 4 use hand-optimized IL assembly to minimize overhead when calling OpenGL functions, and OpenTK 5 is planned to use the new C# function pointers to reduce overhead even further.
+
+We take great pains to make it as efficient as possible.  However, keep in mind that the underlying runtimes (.NET/Mono) use both JIT compilation and garbage collection, which can complicate the definition of “fast.”  Still, we place significant emphasis on performance, so if you believe something could run faster, please [open an issue on our GitHub](https://github.com/opentk/opentk/issues)!
+
+### Does OpenTK support...
+
+#### Vulkan or Metal?
+
+OpenTK does not currently support [Vulkan](https://www.vulkan.org/) or [Metal](https://developer.apple.com/metal/).  OpenTK 5 may not either, but the changes for OpenTK 5 may make it easier to support Vulkan in the future.  Feel free to submit a pull request and [help out](#How-can-I-help)!
+
+#### DirectX or Direct3D?
+
+OpenTK is focused on cross-platform, portable, open standards.  There are other libraries that support DirectX if you want to exclusively target Microsoft platforms.
+
+#### OpenVG? OpenVX? OpenXR?
+
+OpenTK does not currently support the Khronos group’s other open standards.  Feel free to submit a pull request and [help out](#How-can-I-help)!
+
+## Installing and Using OpenTK
+
+### How do I get OpenTK?
+
+OpenTK is distributed as a Nuget package.  The release [Nuget package](https://www.nuget.org/packages/OpenTK/) can be found on [nuget.org](https://nuget.org), and it can be installed via Nuget using your IDE, or via the `nuget` command-line tool.  
+
+The source code is in [our GitHub repository](https://github.com/opentk/opentk).
+
+In Visual Studio, installing OpenTK is as simple as installing any other Nuget package:
+
+- Right-click on your C# project in the Solution Explorer.
+- Choose “Manage Nuget packages.”
+- Type `opentk` into the Search box and press Enter.
+- Click on “**OpenTK** by Team OpenTK.”
+- Click on the “Install” button.
+
+### Which `using` directives do I need?
+
+OpenTK works just like the underlying OpenGL/CL/AL calls, so if you know how to program one of those libraries, you already know most of what you’ll need to know.
+
+First, add a `using` directive for the underlying subsystem you want to use.  For example, you probably want to use one or more of these:
+
+- `using OpenTK.Graphics.OpenGL;`
+- `using OpenTK.Graphics.OpenGL4;`
+- `using OpenTK.Graphics.ES30;`
+- `using OpenTK.Audio.OpenAL;`
+- `using OpenTK.Compute.OpenCL;`
+
+Which ones to use depends on which versions of OpenGL/CL/AL you’re targeting.  We generally recommend with OpenGL that unless you have a specific reason to use another version, you should target `OpenGL4`.
```

Comment:
I think we want to have a clarification here that `OpenGL4` doesn't mean OpenGL version 4 but instead just means that it only exposes the modern functions that you should use, and explaining that it's an unfortunate naming scheme that will be left as is for OpenTK 3 & 4, but will be fixed in OpenTK 5.

Reply:
I changed it to this:

> Which ones to use depends on which flavors of OpenGL/CL/AL you’re targeting.  We generally recommend with OpenGL that you should use `OpenGL4` unless you need the legacy fixed-function pipeline (FFP); the `OpenGL4` namespace works for all OpenGL versions, but it excludes the FFP functions and their overhead.  (In OpenGL 5, we plan to rename these namespaces to make this distinction more obvious.)

Reply:
Good enough for me 🙂